app/models/personal_details.rb

```diff
@@ -1,7 +1,11 @@
 class PersonalDetails < ApplicationRecord
+  # Personal Details
   validates :title, presence: true
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :date_of_birth, presence: true
   validates :nationality, presence: true
+
+  # Contact Details
+  validates :phone_number, :email_address, :address, presence: true, on: :update
```

Comment:
I've found conditional validations - `on: :update` and similar - to be painful in the long run. 

Is there any other way to do this? I guess it's going to be keep being a problem with a multi-stage form.

Reply:
We talked about this irl - going to separate out to different models from here on in, but push ahead and merge this version so we've got something we can show.